src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/migration/PaperApplicationMigration.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.divorce.systemupdate.schedule.migration;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.type.YesOrNo;
+import uk.gov.hmcts.divorce.systemupdate.service.CcdConflictException;
+import uk.gov.hmcts.divorce.systemupdate.service.CcdManagementException;
+import uk.gov.hmcts.divorce.systemupdate.service.CcdSearchCaseException;
+import uk.gov.hmcts.divorce.systemupdate.service.CcdSearchService;
+import uk.gov.hmcts.divorce.systemupdate.service.CcdUpdateService;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.idam.client.models.User;
+
+import static uk.gov.hmcts.divorce.systemupdate.event.SystemMigrateCase.SYSTEM_MIGRATE_CASE;
+
+@Component
+@Slf4j
+public class PaperApplicationMigration implements Migration {
+
+    @Autowired
+    private CcdSearchService ccdSearchService;
+
+    @Autowired
+    private CcdUpdateService ccdUpdateService;
+
+    @Override
+    public void apply(final User user, final String serviceAuthorization) {
+        var isAAT = Boolean.parseBoolean(System.getenv().get("CITIZEN_UPDATE_CASE_STATE_ENABLED"));
```

Comment:
as the flag is not generic I think it's better to use a new one as it is confusing CITIZEN_UPDATE_CASE_STATE_ENABLED. It's best to use APP2OFFLINE_FLAG_MIGRATION_ENABLED or something like that